js/compileShell.py

```diff
@@ -578,6 +578,7 @@ def fixateVer(pcfile):
         """Returns the current version number (47.0a2)."""
         with open(pcfile, 'rb') as f:
             for line in f:
+                line = line.decode("utf-8", "replace")
```

Comment:
What I meant was, I [switched to](https://github.com/MozillaSecurity/funfuzz/pull/119/commits/da871d5e1dabedb989e93e44261946cffd1d7f53) use io.open but with 'rb' mode:

```
with io.open(pcfile, mode='rb', encoding="utf-8", errors="replace")
```